WordPress/src/main/res/values/dimens.xml

```diff
@@ -128,4 +129,19 @@
     <dimen name="stats_button_corner_radius">4dp</dimen>
     <dimen name="stats_barchart_height">128dp</dimen>
 
+    <!-- my site -->
+    <dimen name="my_site_add_button_padding_bottom">5dp</dimen>
```

Comment:
I've combined those `10dp` values as `my_site_margin_general` though I have to say I am not too happy about the naming, but those values are used in quite a bit of places, so I don't know what else we can name it.

I wanted to combine the two `11dp` values, but couldn't come up with something that would make sense. Maybe we can use names like `my_site_margin_11` but that's so weird that I don't think it's worth it.